glean-core/ios/sdk_generator.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+GLEAN_PARSER_VERSION=1.6.0
+
+read -r -d '' RUN_PYTHON_SCRIPT <<'EOF'
```

Comment:
Mh.. I wish we had a way to share this between the gradle plugin and this.

Reply:
Yeah, before we land this in any form that it should be used by others we want to unify this across both Glean SDKs

Reply:
There is a related bug that the Gradle plugin will need to resolve to make this possible: https://bugzilla.mozilla.org/show_bug.cgi?id=1579541